ts/components/wallet/WalletLayout.tsx

```diff
@@ -30,6 +30,21 @@ const styles = StyleSheet.create({
   },
   noBottomPadding: {
     paddingBottom: 0
+  },
+  firstCard: {
+    flex: 1,
+    transform: [{ rotateX: "-40deg" }, { scaleX: 0.98 }]
+  },
+  secondCard: {
+    flex: 1,
+    transform: [
+      { rotateX: "-40deg" },
+      { translateY: -(58 + 20) * Math.sin(40 / 2) * Math.sin(40) },
```

Comment:
🤓

Reply:
Can you elaborate on the reason for using `sin(a/2) * sin(a)` ? 

The way I would expect it to work is the following:

<img width="563" alt="image" src="https://user-images.githubusercontent.com/107715/43206564-5383e93a-9026-11e8-9801-60fc5fee1b20.png">

Where `h` is the height of the "header-only" card (which, btw, I would recommend exporting to a constant since it's being used both there and in `ts/components/card/index.ts`, and `alpha` is the rotation angle (40) (which I would also recommend exporting to a constant). Then `delta` (in (II)) is an offset that would shift the card down a notch to make the "cards stacking" look better

Reply:
The math doesn't evaluate the height of the component but the delta value, which depends on both the angle and the real height of the card. If both the values will be constant also in future development (angle at 40 and height of the flat card equal to 58) that math could be substituted with a constant. At the contrary, if you think the card height could became a relative value, it could be preferred to maintain the math to avoid future unwanted effects.

Reply:
I'm still unclear about the math you used in the code, or how that translates to `1-cos(a)` -- can you give me a rundown of it?

As for the values, I would recommend using constants for the card height and the angle, as they may -- at some point -- be subject to change, as much of the code as possible should update automatically